tasks/global-setup.yml

```diff
@@ -7,3 +7,14 @@
     state: present
     backrefs: yes
   notify: restart_gitlab_runner
+
+- name: Add sentry dsn to config
+  lineinfile:
+    dest: /etc/gitlab-runner/config.toml
+    regexp: '^sentry_dsn ='
+    line: 'sentry_dsn = "{{ sentry_dsn }}"'
```

Comment:
The current variable format is `gitlab_runner_*` so something like `gitlab_runner_sentry_dns` would be more conform the rest. Since it is also a new value, add it to the readme too.

Other then that i don't see a problem with adding this.

Reply:
Yep, that makes sense. I'll do that.

Reply:
Yes, might be good to have it in defaults too as you suggest just so people who look there see that it's an option.